app/controllers/integrations/github_controller.rb

```diff
@@ -11,8 +11,23 @@ def self.secret_token
     ENV['GITHUB_HOOK_SECRET']
   end
 
+  def create
+    if github_event_type == "status"
+      handle_commit_status_event
+
+      render json: {deploy_ids: [], messages: "status event received"}, status: :ok
```

Comment:
maybe call super here ... to not break existing behavior / not have to copy-paste the rendering ?

Reply:
```Ruby
handle_commit_status_event  if github_event_type == "status"
super
```